assets/base.css

```diff
@@ -2183,8 +2183,7 @@ input[type='checkbox'] {
 }
 
 .announcement-bar__message {
-  text-align: center;
-  padding: 1rem 2rem;
+  padding: 1rem 0;
```

Comment:
We should probably make sure the `.announcement-bar__link` gets the same treatment [when using links](https://user-images.githubusercontent.com/28404165/173389567-74ca66ec-01f4-4bd3-a11c-c5eb63ec07cb.png) for the announcement text:

**Left aligned announcement text with link**

<img src="https://user-images.githubusercontent.com/28404165/173388377-36e7a084-53c9-493f-93d5-8c451c535140.png" width="500" height="auto">

**Right aligned announcement text with link**

<img src="https://user-images.githubusercontent.com/28404165/173388537-9a95b440-33e4-4598-bfe7-09b01729db81.png" width="500" height="auto">